src/index.js

```diff
@@ -22,5 +22,6 @@ import Parser from './Parser.js';
 import Plugin from './Plugin.js';
 import Result from './Result.js';
 import Rule from './Rule.js';
+import * as utils from './utils.js';
 
-export { Formatter, Parser, Plugin, Result, Rule };
\ No newline at end of file
+export { Formatter, Parser, Plugin, Result, Rule, utils };
```

Comment:
This will export all the utility functions in a "utils" name space won't it? Can we export them individually? Otherwise, you would have to call it like this:

```javascript
import { utils } from 'i18nlint-common';

...
highlight: `<e0>${utils.withVisibleWhitespace(str)}</e0>` +
```

instead of:

```javascript
import { withVisibleWhitespace } from 'i18nlint-common';

...
highlight: `<e0>${withVisibleWhitespace(str)}</e0>` +
```

Reply:
I figured it might make more sense to keep utility functions grouped together - distinct from "core" abstract classes. I'll change it

Reply:
Changed in ea53e55